gdal/frmts/netcdf/netcdfdataset.cpp

```diff
@@ -3368,6 +3368,17 @@ void netCDFDataset::SetProjectionFromVar( int nGroupId, int nVarId,
             // oSRS.SetWellKnownGeogCS("WGS84");
         }
     }
+    else
+    {
+        // Dataset from https://github.com/OSGeo/gdal/issues/4075 has a "crs"
```

Comment:
I wouldn't worry about reading in the non CF-compliant CRS information. That isn't something that has worked in the past and is handled by rioxarray: https://corteva.github.io/rioxarray/stable/getting_started/crs_management.html. Definitely want to encourage others to use the CF-compliant version for storing the CRS.

Reply:
ok thaks for the info. I'll keep that as it's now coded

Reply:
Okay, I would use fromUserInput then as I have seen some CRS attrs with WKT strings as well.

Reply:
Reference: https://github.com/opendatacube/datacube-core/blob/3997b90f7cc1fe909b3e5ee0da395bb07fd969d5/datacube/api/core.py#L514

Reply:
Looks like it could be an EPSG reference as well: https://github.com/opendatacube/datacube-core/blob/3997b90f7cc1fe909b3e5ee0da395bb07fd969d5/datacube/utils/geometry/_base.py#L134